sponsorship_compassion/mappings/res_partner_mapping.py

```diff
@@ -32,7 +32,9 @@ class ResPartnerMapping(OnrampMapping):
     }
 
     FIELDS_TO_SUBMIT = {
-        'Gender': lambda gender: 'Female' if gender == 'F' else 'Male',
+        'Gender': lambda title:
+            title.gender if title in
```

Comment:
```suggestion
            title.gender if title.with_context(lang='en_US').name in
```
I think title is a record so title.name should be used to check your inclusion. I also included the language force to make sure we are comparing the english names.